### PR TITLE
Clarify Sass integration

### PR DESCRIPTION
**Pull request by lamosty**
Enhances #1404

### CODE REVIEW

**Comment by retlehs** [comment]
thanks for this. to be consistent with the rest of the repo and site, please change "LESS" to "Less"

File: assets/styles/main.scss.example [hunk 1]
```diff
@@ -5,6 +5,7 @@
 // 3. Rename this file to `main.scss` and remove `main.less`
 // 4. Rename `editor-style.less` to `editor-style.scss`
 // 5. Update the `assets/manifest.json` styles dependencies from `.less` to `.scss`
+// 6. Refactor variables from LESS to Sass (https://discourse.roots.io/t/regarding-the-grid-help-translating-less-to-sass/3253)
```
Comment (swalkinshaw) [inline assets/styles/main.scss.example]: Nitpick but can you change "Refactor" to just "Change"

**Comment by lamosty** [comment]
@retlehs Sorry, haven't noticed.

@swalkinshaw As you wish :)

**Comment by JulienMelissas** [comment]
@lamosty - one last thing - can you please rebase and squash your commits?

**Pull request merged**